README.md

```diff
@@ -5,33 +5,42 @@
 [![Docker Pulls](https://img.shields.io/docker/pulls/frameworkcontroller/frameworkcontroller.svg)](https://hub.docker.com/u/frameworkcontroller)
 [![License](https://img.shields.io/github/license/microsoft/frameworkcontroller.svg)](https://github.com/microsoft/frameworkcontroller/blob/master/LICENSE)
 
-As one standalone component of [Microsoft OpenPAI](https://github.com/microsoft/pai), FrameworkController is built to orchestrate all kinds of applications on [Kubernetes](https://kubernetes.io) by a single controller.
+As one standalone component of [Microsoft OpenPAI](https://github.com/microsoft/pai), FrameworkController (FC) is built to orchestrate all kinds of applications on [Kubernetes](https://kubernetes.io) by a single controller, especially for DeepLearning applications.
 
 These kinds of applications include but not limited to:
-- __Stateless and Stateful Service__ (Nginx, TensorFlow Serving, HBase, Kafka, etc)
-- __Stateless and Stateful Batch__ (KD-Tree Building, Batch Data Processing, etc)
-- __Any combination of above applications__ (Distributed TensorFlow Training, Stream Data Processing, etc)
+- __Stateless and Stateful Service__:
+  - DeepLearning Serving: [TensorFlow Serving](https://www.tensorflow.org/tfx/guide/serving), etc.
+  - Big Data Serving: HDFS, HBase, Kafka, Nginx, etc.
+- __Stateless and Stateful Batch__:
+  - DeepLearning MPI Training: [Horovod Training](https://horovod.readthedocs.io), etc.
```

Comment:
```suggestion
  - DeepLearning MPI Training: [Horovod Distributed Training](https://horovod.readthedocs.io/en/stable/mpirun.html), etc.
```

Reply:
OK

Reply:
BTW, does all Horovod Training can be considered as MPI based Training (no need param server, and only need a single work task role)?

Reply:
no, there's single GPU training, and besides mpi for distributed training, Horovod also supports other communication library like [Gloo](https://github.com/horovod/horovod#gloo)